sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/manager/targeting/TargetingContextAccessor.java

```diff
@@ -2,17 +2,15 @@
 // Licensed under the MIT License.
 package com.azure.spring.cloud.feature.manager.targeting;
 
-import reactor.core.publisher.Mono;
-
 /**
  * Interface for providing a Targeting Context to the `Microsoft.TargetingFilter`.
  */
-public interface ITargetingContextAccessor {
+public interface TargetingContextAccessor {
 
     /**
-     * Returns the Targeting Context for Feature Targeting evaluation.
-     * @return Mono{@literal <}TargetContext{@literal >}
+     * Configures the Targeting Context for Feature Targeting evaluation.
+     * @param context Targeting Context for selecting feature
      */
-    Mono<TargetingContext> getContextAsync();
+    void getContextAsync(TargetingContext context);
```

Comment:
How about renaming to `configureTergetingContextAsync`?

Reply:
I was trying to limit the number of breaking change the method has been out like that for at least 2ish years.

Reply:
OK.

Reply:
nit:
1. We already updated the interface name. It already a breaking change. Change method is not a big change comparing to change interface name.
2. `getContextAsync` will make customer confused that it returns `void` but havs `get` in the method name.

Of course, you can decide by yourself. I just give some suggestion.